src/containers/AccountModal/AccountModal.tsx

```diff
@@ -28,7 +28,7 @@ const AccountModal = () => {
   const viewParam = useQueryParam('u');
   const [view, setView] = useState(viewParam);
   const message = useQueryParam('message');
-  const { loading, auth } = AccountStore.useState((s) => s);
+  const { loading, auth } = useAccountStore();
```

Comment:
Isn't destructuring from the function more performant?
```suggestion
  const [loading, auth] = useAccountStore((s) => [s.loading, s.auth]);
```

Reply:
I doubt there's much performance difference with any of these solutions. My vote would be for object destructuring over array destructuring. It's a bit safer in getting the right properties to the right variables. With the array style, if you have 2 properties of the same type, it's easier to mix them up and not notice.

Reply:
Yes, agreed!